lua/toggleterm/commandline.lua

```diff
@@ -41,6 +41,8 @@ function M.parse(args)
         local key, value = arg[1], arg[2]
         if key == "size" then
           value = tonumber(value)
+        elseif key == "go_back" then
+          value = value ~= "false" and value ~= "0"
```

Comment:
You can just stick with `0|1` as I mentioned above I might change it but it's a minor point in the grand scheme of things